pkgs/top-level/all-packages.nix

```diff
@@ -42098,4 +42098,6 @@ with pkgs;
   code-maat = callPackage ../development/tools/code-maat {};
 
   mdhtml = callPackage ../tools/text/mdhtml { };
+
+  revolver = callPackage ../development/tools/revolver { };
```

Comment:
please use `pkgs/by-name` and remove this line

Reply:
i figured it was used like a library by other scripts so I placed it under pkgs/development/tools